config/cluster-api-provider-aws/kustomization.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: capa-system
+
+configurations:
+- namingConfiguration.yaml
+
+resources:
+- base/core-components.yaml
+
+patches:
+- patch: |-
+    - op: add
+      path: /metadata/annotations/service.beta.openshift.io~1serving-cert-secret-name
+      value: capa-webhook-service-cert
+  target:
+    kind: Service
+    name: capa-webhook-service
+#- patch: |-
+#    - op: add
+#      path: /metadata/annotations/service.beta.openshift.io~1inject-cabundle
+#      value: true
+#    - op: delete
+#      path: /metadata/annotations/cert-manager.io~1inject-ca-from
+#  target:
+#    annotationSelector: "cert-manager.io/inject-ca-from=capa-system/capa-serving-cert"
+- patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/command
+      value:
+      - /bin/cluster-api-provider-aws-controller-manager
+    - op: replace
+      path: /spec/template/spec/containers/0/image
+      value: registry.redhat.io/openshift4/ose-aws-cluster-api-controllers-rhel9:__.Values.manager.image.tag__
```

Comment:
Why do you patch the charts values with _ _ .Values.._ _ and then uses the script to replace __ with {{ ? 
if you set the value as `value: 'registry.redhat.io/openshift4/ose-aws-cluster-api-controllers-rhel9:{{ .Values.manager.image.tag }}' ` OR better `value: '{{ .Values.manager.image.url }}:{{ .Values.manager.image.tag }}' ` it should works with helm chart .

Reply:
It was used for  more general cases.

I replaced the placeholder with final value without `{{ .Values.manager.image.url }}` due to a previous request to have the variable only for the image version. Isn't it?